src/constants/oracleConfig.ts

```diff
@@ -20,81 +10,252 @@ const optimisticV2Chains: Record<
   oracle.types.state.PartialChainConfig
 > = {};
 
-// enable local node if debug is on
+// enable local node if debug is on, this only works as a fork of mainnet
 if (process.env.REACT_APP_DEBUG) {
-  optimisticChains[mmChainId] = {
+  const chainId = ChainId.MM_TESTNET;
+  const chain = CHAINS[chainId];
+  const config: oracle.types.state.PartialChainConfig = {
     rpcUrls: ["http://127.0.0.1:8545"],
-    nativeCurrency: CHAINS[ethChainId].nativeCurrency,
-    blockExplorerUrls: [CHAINS[ethChainId].explorerUrl],
-    chainName: CHAINS[ethChainId].name,
+    nativeCurrency: chain.nativeCurrency,
+    blockExplorerUrls: [chain.explorerUrl],
+    chainName: chain.name,
+  };
+  optimisticChains[chainId] = {
+    ...config,
     multicall2Address: "0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696",
     optimisticOracleAddress: "0xC43767F4592DF265B4a9F1a398B97fF24F38C6A6",
   };
+  skinnyChains[chainId] = {
+    ...config,
+    multicall2Address: "0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696",
+    optimisticOracleAddress: "0xeE3Afe347D5C74317041E2618C49534dAf887c24",
+  };
+  optimisticV2Chains[chainId] = {
+    ...config,
+    multicall2Address: "0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696",
+    optimisticOracleAddress: "0xA0Ae6609447e57a42c51B50EAe921D701823FFAe",
+  };
 }
 
 if (process.env.REACT_APP_PROVIDER_URL_1) {
-  const chainConfig: oracle.types.state.PartialChainConfig = {
+  const chainId = ChainId.MAINNET;
+  const chain = CHAINS[chainId];
+  const config: oracle.types.state.PartialChainConfig = {
     rpcUrls: [process.env.REACT_APP_PROVIDER_URL_1],
-    nativeCurrency: CHAINS[ethChainId].nativeCurrency,
-    chainName: CHAINS[ethChainId].name,
-    blockExplorerUrls: [CHAINS[ethChainId].explorerUrl],
+    nativeCurrency: chain.nativeCurrency,
+    chainName: chain.name,
+    blockExplorerUrls: [chain.explorerUrl],
+  };
+  optimisticChains[chainId] = {
+    ...config,
+    optimisticOracleAddress: "0xc43767f4592df265b4a9f1a398b97ff24f38c6a6",
+  };
+  skinnyChains[chainId] = {
+    ...config,
+    optimisticOracleAddress: "0xeE3Afe347D5C74317041E2618C49534dAf887c24",
+  };
+  optimisticV2Chains[chainId] = {
+    ...config,
+    optimisticOracleAddress: "0xA0Ae6609447e57a42c51B50EAe921D701823FFAe",
+  };
+}
+
+if (process.env.REACT_APP_PROVIDER_URL_5) {
+  const chainId = ChainId.GOERLI;
+  const chain = CHAINS[chainId];
+  const config: oracle.types.state.PartialChainConfig = {
+    rpcUrls: [process.env.REACT_APP_PROVIDER_URL_5],
+    nativeCurrency: chain.nativeCurrency,
+    chainName: chain.name,
+    blockExplorerUrls: [chain.explorerUrl],
+  };
+  // no skinny deployment on goerli
+  optimisticChains[chainId] = {
+    ...config,
+    optimisticOracleAddress: "0x6f26Bf09B1C792e3228e5467807a900A503c0281",
+  };
+  optimisticV2Chains[chainId] = {
+    ...config,
+    optimisticOracleAddress: "0x3C8a21099C202003Ec6f050Eb24F8f24a3828Ad3",
```

Comment:
The newly deployed address is `0xA5B9d8a0B0Fa04Ba71BDD68069661ED5C0848884`.

My older PR was closed in favor of https://github.com/UMAprotocol/protocol/commit/f65eb4a256cc36c170af940cafa6ac7d6cae8fb1